client-ts/Microsoft.AspNetCore.SignalR.Client.TS/HttpClient.ts

```diff
@@ -34,19 +34,11 @@ export class HttpClient implements IHttpClient {
                     resolve(xhr.response);
                 }
                 else {
-                    reject({
-                        status: xhr.status,
-                        statusText: xhr.statusText
-                    });
+                    reject(new Error(xhr.statusText));
                 }
             };
 
-            xhr.onerror = () => {
-                reject({
-                    status: xhr.status,
-                    statusText: xhr.statusText
-                });
-            };
+            xhr.onerror = () => reject(new Error(xhr.statusText));
```

Comment:
It would be nice to have some tests for these but I am not sure how hard it would be to add them...